Quadriga/src/main/java/edu/asu/spring/quadriga/domain/impl/ConceptStats.java

```diff
@@ -0,0 +1,112 @@
+package edu.asu.spring.quadriga.domain.impl;
+
+import java.util.Collections;
+import java.util.Comparator;
+import java.util.List;
+
+import org.springframework.stereotype.Service;
+
+import edu.asu.spring.quadriga.domain.IConceptStats;
+import edu.asu.spring.quadriga.domain.conceptcollection.IConceptCollectionConcepts;
+
+/**
+ * This class represents one concept with limited properties. It provides
+ * existing properties of concept in terms of concept id, description and lemma
+ * of the concept. Apart from that, it also provides frequency of concept
+ * appears in a certain object.
+ *
+ * @author ajaymodi
+ *
+ */
+
+@Service
+public class ConceptStats implements IConceptStats

```

Comment:
This should not be a service. It's just for information holding, right? So you want as many as you need, not just one.